<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CelestianClass</class>
 <widget class="QMainWindow" name="CelestianClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Celestian</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="practice">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动修仙</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pack">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>丹药背包操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="act">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>采药</string>
    </property>
   </widget>
   <widget class="QPushButton" name="communityTask">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>获取宗门任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dailyTask">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>每日悬赏令</string>
    </property>
   </widget>
   <widget class="QPushButton" name="market">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>坊市操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyPotuon">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>快速购买当前突破丹药</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bank">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>灵庄操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signIn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>签到</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusIndicator">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>系统运行状态</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Celestian.qrc"/>
 </resources>
 <connections/>
</ui>
